### DISCUSSION TITLE
Test Application Pass when copy to apache is not start Please help

### DISCUSSION THREAD

**Discussion by kenakung** [netbox-community/netbox#11849] [General]
I'm copy to Apache is try runing apache2 is error when i'm remove it's can runing normally

[![ezOGRS.md.png](https://sv1.picz.in.th/images/2023/02/28/ezOGRS.md.png)](https://www.picz.in.th/image/1.ezOGRS)

i'm run test application is runing normally

[![ezOJun.md.png](https://sv1.picz.in.th/images/2023/02/28/ezOJun.md.png)](https://www.picz.in.th/image/2.ezOJun)

[![ezOLMg.md.png](https://sv1.picz.in.th/images/2023/02/28/ezOLMg.md.png)](https://www.picz.in.th/image/3.ezOLMg)

Please help

**Comment by candlerb** [comment]
(Aside: please don't use screenshots, and please don't host them on a third-party site.  It's *much* easier to read if you just copy-paste the text from your console)

After logging in to your image hosting site, I can see that the first image tells you exactly what to do: it tells you to run `journalctl -xe` (or better, `journalctl -eu apache2`) to see the errors generated by Apache.  You can also try `tail /var/log/apache2/error.log` or similar - it may be in a different location depending on which OS you're using.

Also: the error is clearly in your "netbox.conf", which you've demonstrated by removing it and restarting Apache.  Therefore, you need to show us the content of this file, if you want help identifying the problem.
